server/src/main/java/io/spine/server/event/EventStore.java

```diff
@@ -310,4 +352,50 @@ public ServerServiceDefinition build() {
             return result;
         }
     }
+
+    /*
+     * Logging methods
+     *******************/
+
+    private void logStored(Event request) {
+        if (logger == null) {
+            return;
+        }
+        if (logger.isTraceEnabled()) {
+            logger.trace("Stored: {}", TextFormat.shortDebugString(request));
+        }
+    }
+
+    private void logStored(Iterable<Event> events) {
+        for (Event event : events) {
+            logStored(event);
+        }
+    }
+
+    private void logReadingStart(EventStreamQuery request, StreamObserver<Event> responseObserver) {
+        if (logger == null) {
+            return;
+        }
+
+        if (logger.isInfoEnabled()) {
+            final String requestData = TextFormat.shortDebugString(request);
+            logger.info("Creating stream on request: {} for observer: {}",
+                        requestData,
+                        responseObserver);
+        }
+    }
+
+    /** Returns default logger for the class. */
+    public static Logger log() {
```

Comment:
As the doc states, this is the default logger for this class. It's possible to create an `EventStore` with the custom logger or with no logger at all. 
This method is used in the `EventBus`.

BTW, this "change" is just a result of merging with some severe changes in this file.